.github/workflows/build.yaml

```
@@ -63,16 +77,53 @@ jobs:
+    uses: ./.github/workflows/manifest.yaml
     secrets: inherit
     permissions: &permissions
       contents: write
       actions: write
       packages: write
       security-events: write
+
+  sync:
+    name: Sync repositories
+    needs: manifest
+    uses: ./github/workflows/sync.yaml
```

actionlint (1.7.8)

could not read reusable workflow file for "./github/workflows/sync.yaml": open /home/jailuser/git/github/workflows/sync.yaml: no such file or directory

(workflow-call)